README.md

```diff
@@ -1,5 +1,55 @@
-# ECL Twig Components
+# ECL-Twig Library
 
-Twig components for the EC new visual identity (EU coming soon).
+[![Build Status](https://drone.fpfis.eu/api/badges/ec-europa/ecl-twig/status.svg)](https://drone.fpfis.eu/ec-europa/ecl-twig)
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lernajs.io/)
 
-[Read the docs](./docs).
+The ECL-Twig is a library of [twig](https://twig.symfony.com/) templates, implementing the Europa Component Library (ECL) vanilla components.
+ECL is a library of components applicable to all European Commission websites (hosted under europa.eu and ec.europa.eu domains), the library contains all the available components to build your website.
+
+For information on how to use the ECL library please refer to the information available in the [ECL docs](https://github.com/ec-europa/europa-component-library/blob/v2-dev/docs/README.md)
+
+The twig implementation of the ECL components is meanly meant to serve [Drupal 8](http://drupal.org) applications, therefore the version of twig in use is "1.x".
+
+## EC/EU
+
+ECL is a library used both for EC (European Commission) and EU (European Union) websites, it comes indeed with two sets of distributes resources (css, js) for the two institutions.
+
+Although they sometimes differ in style, in the current v2 version, the html markup is just the same, ECL-Twig is shipping one single component that can be used both while working on an EC or EU website, indeed, the template is not aware of the "system" in use, it doesn't need to be.
```

Comment:
```suggestion
Although they may differ in style, in the current v2 version, the HTML markup is the same. ECL-Twig is shipping one single component that can be used while working on both EC and EU website. Templates are not system-aware because they don't need to be.
```